tpot/tpot_mdr_classifier.py

```diff
@@ -0,0 +1,30 @@
+import numpy as np
+
+tpot_mdr_classifier_config_dict = {
+
+    # Classifiers
+    
+    'mdr.MDR': {
+    'tie_break': tie_break % 2
+    'default_label': default_label % 2
+    },
+    
+    # Feature Selectors
+    
+    'skrebate.ReliefF': {
+        'n_features_to_select': max(1, n_features_to_select)
```

Comment:
The ReliefF algorithm should also include the `n_neighbors parameter`, with the values

```[2, 10, 50, 100, 250, 500]```